frontend/src/mixins/formComponentPropsMixin.js

```diff
@@ -46,4 +49,15 @@ export const formComponentPropsMixin = {
       default: () => [],
     },
   },
+  computed: {
+    defaultLabel() {
+      if (this.label !== undefined) {
+        return this.label
+      }
+      if (!this.entityName || !this.name) {
```

Comment:
Should we output a warning somewhere when this happens?

Reply:
No, because from my view `entityName` and field `name` are like an opt in into this functionality.